CMakeLists.txt

```diff
@@ -44,12 +44,12 @@ set(LIBRARY_OUTPUT_PATH "${CMAKE_SOURCE_DIR}/build")
 file(GLOB_RECURSE COMMON_SRC "common/src/*.cpp")
 set(COMMON_LIBRARY "Common")
 
-add_library(${COMMON_LIBRARY} STATIC ${COMMON_SRC})
+add_library(${COMMON_LIBRARY} STATIC ${COMMON_SRC} editor/include/editor/ParametersWindow.h)
```

Comment:
To remove - unfortunately CLion adds this automatically. Maybe it can be changed in settings.